src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Observability Platform API"
+linkTitle: Observability Platform API
+description: Documentation on the observability-platform API architecture deployed and maintained by Giant Swarm.
+weight: 80
+menu:
+  principal:
+    identifier: overview-observability-platform-api
+    parent: overview-observability
```

Comment:
I would actually add it not to the overview section, but to the tutorial section, because it is quite a niché functionality that needs a good maturity in observability. I wouldn't want to advertise it when one just wants to get an overview. So let's add it to tutorials/observability - what do you think?